--- sddm-0.19.0/services/CMakeLists.txt~	2022-01-19 15:26:03.000000000 +0000
+++ sddm-0.19.0/services/CMakeLists.txt	2022-01-19 15:26:56.010809938 +0000
@@ -12,16 +12,16 @@
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/sddm-greeter.pam.in" "${CMAKE_CURRENT_BINARY_DIR}/sddm-greeter.pam")
 
 if(EXISTS "/etc/debian_version")
-    install(FILES debian.sddm-autologin.pam DESTINATION ${CMAKE_INSTALL_FULL_SYSCONFDIR}/pam.d RENAME sddm-autologin)
-    install(FILES debian.sddm-greeter.pam DESTINATION ${CMAKE_INSTALL_FULL_SYSCONFDIR}/pam.d RENAME sddm-greeter)
-    install(FILES debian.sddm.pam DESTINATION ${CMAKE_INSTALL_FULL_SYSCONFDIR}/pam.d RENAME sddm)
+    install(FILES debian.sddm-autologin.pam DESTINATION /usr/share/pam.d RENAME sddm-autologin)
+    install(FILES debian.sddm-greeter.pam DESTINATION /usr/share/pam.d RENAME sddm-greeter)
+    install(FILES debian.sddm.pam DESTINATION /usr/share/pam.d RENAME sddm)
 else()
     if(HAVE_PAM_FAILLOCK)
-        install(FILES sddm-autologin.pam DESTINATION ${CMAKE_INSTALL_FULL_SYSCONFDIR}/pam.d RENAME sddm-autologin)
+        install(FILES sddm-autologin.pam DESTINATION /usr/share/pam.d RENAME sddm-autologin)
     else()
-        install(FILES sddm-autologin-tally2.pam DESTINATION ${CMAKE_INSTALL_FULL_SYSCONFDIR}/pam.d RENAME sddm-autologin)
+        install(FILES sddm-autologin-tally2.pam DESTINATION /usr/share/pam.d RENAME sddm-autologin)
     endif()
 
-    install(FILES sddm.pam DESTINATION ${CMAKE_INSTALL_FULL_SYSCONFDIR}/pam.d RENAME sddm)
-    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/sddm-greeter.pam" DESTINATION ${CMAKE_INSTALL_FULL_SYSCONFDIR}/pam.d RENAME sddm-greeter)
+    install(FILES sddm.pam DESTINATION /usr/share/pam.d RENAME sddm)
+    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/sddm-greeter.pam" DESTINATION /usr/share/pam.d RENAME sddm-greeter)
 endif()
