From 5d2a0b509122e4fb5be08d63d141d1a867c93267 Mon Sep 17 00:00:00 2001
From: Thiago Macieira <thiago.macieira@intel.com>
Date: Wed, 18 Jul 2018 22:48:28 -0700
Subject: [PATCH] Install the PAM config files where Clear expects them to be

Signed-off-by: Thiago Macieira <thiago.macieira@intel.com>
---
 services/CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/services/CMakeLists.txt b/services/CMakeLists.txt
index fbf7608..9bcdb27 100644
--- a/services/CMakeLists.txt
+++ b/services/CMakeLists.txt
@@ -10,6 +10,6 @@ else()
 endif()
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/sddm-greeter.pam.in" "${CMAKE_CURRENT_BINARY_DIR}/sddm-greeter.pam")
 
-install(FILES sddm.pam DESTINATION ${CMAKE_INSTALL_FULL_SYSCONFDIR}/pam.d RENAME sddm)
-install(FILES sddm-autologin.pam DESTINATION ${CMAKE_INSTALL_FULL_SYSCONFDIR}/pam.d RENAME sddm-autologin)
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/sddm-greeter.pam" DESTINATION ${CMAKE_INSTALL_FULL_SYSCONFDIR}/pam.d RENAME sddm-greeter)
+install(FILES sddm.pam DESTINATION /usr/share/pam.d RENAME sddm)
+install(FILES sddm-autologin.pam DESTINATION /usr/share/pam.d RENAME sddm-autologin)
+install(FILES "${CMAKE_CURRENT_BINARY_DIR}/sddm-greeter.pam" DESTINATION /usr/share/pam.d RENAME sddm-greeter)
-- 
2.18.0

